### ISSUE TITLE
RegressionTables v0.0.2 is incompatible with DataFrames v0.11.1

### ISSUE THREAD

**Issue by ltf-canada** [jmboehm/RegressionTables.jl#4]
Installing RegressionTables on Julia v0.6.1 forces a DataFrames downgrade to v0.10.1.

**Comment by floswald** [comment]
from the REQUIRE it seems that this is caused by another dependency (distributions?). there are many packages out there not yet on DataFrames v0.11.1. it's going to be some work to figure out which ones. That said, @jmboehm why do you require `Distributions 0.4.6`? i see that's from march 2014?

**Comment by floswald** [comment]
oh i just learned that `Distributions 0.4.6` means that you have to have *at least* that version. still, i can only get the new dataframes working in a completely separate `JULIA_PKGDIR`. there's just too many old dependencies around still.

**Comment by floswald** [comment]
i think this is because of the dependency on https://github.com/matthieugomez/FixedEffectModels.jl/issues/21

**Comment by floswald** [comment]
the latest release doesn't work for me yet:

```
└─version range [0.9.0,0.11.0) required by package FixedEffectModels, whose allowed version range is [0.4.0,∞):
  └─version range [0.4.0,∞) required by package RegressionTables, whose allowed version range is [0.0.0-,∞):
    └─[see above for RegressionTables backtrace]
```

maybe you have to explicitly require FixedEffectModels 0.5.0 in your REQUIRE?

**Comment by floswald** [comment]
hmm, it's odd because I can see that travis installs FixedEffectModels 0.5.0 for you and all is fine.

**Comment by jmboehm** [comment]
Yeah, I also sometimes run into these issues... try removing & freeing both and then adding them, for some reason this sometimes works for me.

**Comment by jmboehm** [comment]
I'm now requiring FixedEffectModels 0.5.0, let me know whether this helps.

**Comment by jmboehm** [comment]
Fixed in commit c0de1336ee90899152f1a65beee449611a0495ab
